README.md

```diff
@@ -65,6 +66,7 @@ The _"non-endorsed"_ status means that the plugin is not endorsed by the origina
 | [**shared_preferences_tizen**](packages/shared_preferences) | ✔️ | ✔️ | ✔️ | ✔️ |
 | [**sqflite_tizen**](packages/sqflite) | ✔️ | ✔️ | ✔️ | ✔️ |
 | [**tizen_app_control**](packages/tizen_app_control) | ✔️ | ✔️ | ✔️ | ✔️ |
+| [**tizen_log**](packages/tizen_log) | ✔️ | ✔️ | ✔️ | ✔️ |
```

Comment:
I don't think this plugin is compatible with TV devices (including TV emulator).

Reply:
I found [this instruction](https://developer.samsung.com/smarttv/develop/extension-libraries/smart-view-sdk/receiver-apps/debugging.html) to enable debugging on a real TV. Unfortunately, I am not able to check if it gives an access to logs. On an emulator I cannot get logs indeed.

Reply:
I'm pretty sure that there's no way to get dlogs from TV devices (unless you're using a development board). Web apps use their own logging mechanism. .NET apps have no logging support on TV.